molecule/ubuntu-16.04/playbook.yml

```diff
@@ -22,3 +22,12 @@
           ajp_port: 8009
           wars:
             - url: https://tomcat.apache.org/tomcat-6.0-doc/appdev/sample/sample.war
+          roles:
```

Comment:
This is nice stuff. Good suggestion, well defined in these variables.

I'll see if I can merge just these changes, if that's not possible I'll ask you to make another merge request just for this feature.

Reply:
I actually did this cause I'm trying to debug the server I'm building to figure out why tomcat is starting, but my app doesn't seem to be. Eventually gonna wire up LDAP in there, once I figure out how.